tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -128,6 +134,9 @@ class DashboardPage extends BasePage {
         chartGridItemContainer: `${slct(COMMON_DASH_SELECTORS.DASH_GRID_ITEM)} .chartkit`,
         dashPluginWidgetBody: slct('chart-widget'),
         dashkitGridItem: slct('dashkit-grid-item'),
+
+        chartResetButton: '.widget-header__filters-controls .yc-button',
+        yfmContentWrapper: slct('yfm-wrapper-html'),
```

Comment:
For this can you please create and use
```
export const enum YfmQa {
    WrapperHtml = 'yfm-wrapper-html',
}
```
in `src/shared/constants/qa/components.ts`?